app/screens/home/bottomTabs/Home.tsx

```diff
@@ -522,10 +530,91 @@ const Home = ({ route, navigation }: any): any => {
                 onChange={ondobChange2}
               />
             )}
-            <DailyReads />
-            <FeatureDivideArea>
+             <SearchBox>
+              <OuterIconRow>
+
+                <Pressable style={styles.pressablePadding} onPress={async (e): Promise<any> => {
+                  e.preventDefault();

```

Comment:
There's no default behavior to prevent for Pressable. You can remove preventDefault().
https://github.com/adobe/react-spectrum/issues/963